spec.yaml

```diff
@@ -89,10 +89,16 @@ paths:
                 "code": "RESOURCE_CREATED",
                 "message": "A new cluster has been created with ID 'wqtlq'"
               }
-        default:
-          description: Error
+        "401":
+          description: Permission Denied
```

Comment:
I dont really understand the point of 'default'.. why is it better? how does not using it force us to specify every single possible response code?